nin/frontend/app/scripts/components/main.js

```diff
@@ -27,13 +27,10 @@ class Main extends React.Component {
       fullscreen: false,
       showFramingOverlay: false,
       mute: localStorage.getItem('nin-mute') ? true : false,
-      volume: 1,
       globalJSErrors: {},
     };
 
-    if (localStorage.hasOwnProperty('nin-volume')) {
-      this.state.volume = +localStorage.getItem('nin-volume');
-    }
+    demo.music.setVolume(1 - this.state.mute);
```

Comment:
I use negation to toggle the boolean mute flag, while setVolume takes a number between 0 and 1. I suppose use negation will technically work here too, since true is 1 and false is 0, but I felt it was better to actually pass a number instead.